sys/include/fs/littlefs_fs.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2017 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_littlefs  littlefs integration
+ * @ingroup     pkg_littlefs
+ * @brief       RIOT integration of littlefs
+ *
+ * Integration of littlefs into RIOT
```

Comment:
sounds like what `@brief` is already telling 😉

Reply:
Removed